codegens/curl/lib/index.js

```diff
@@ -186,6 +186,15 @@ self = module.exports = {
         description: 'Set a character used to mark the continuation of a statement on the next line ' +
           '(generally, \\ for OSX/Linux, ^ for Windows)'
       },
+      {
+        name: 'Quote Type',
+        id: 'quoteType',
+        availableOptions: ['\'', '"'],
+        type: 'enum',
+        default: '\'',
+        description: 'Set a Quote type to be used(Single or Double) to be used,' +
```

Comment:
This copy needs to be looked at. I'd suggest using https://www.grammarly.com/ to proofread any copy. Also, does this need to be different from what's mentioned in the readme?

Reply:
Fixed this